src/vuestic-theme/vuestic-components/va-checkbox/va-checkbox-docs.md

```diff
@@ -13,13 +13,19 @@
   :errorCount="3"
   label="With errors"
 />
-```  
+<va-checkbox
+  v-model="array"
+  array-value="one"
+/>
+```
 
 **Props**
 * `id` - String
 * `name` - String
 * `label` - String - label to the right of checkbox
 * `value` - Boolean - main value
+* `array` - Array<String>
```

Comment:
No such a prop.

Reply:
Sorry. I didn't understand correctly.
I'll fix this.